code/modules/mob/living/simple_animal/hostile/mimic.dm

```diff
@@ -266,3 +270,22 @@ GLOBAL_LIST_INIT(protected_objects, list(/obj/structure/table, /obj/structure/ca
 		return
 	icon_state = TrueGun.icon_state
 	icon_living = TrueGun.icon_state
+
+/datum/action/innate/mimic/root

```

Comment:
this shouldn't be ```/innate```

Reply:
Okay, what types of actions are supposed to be innate?